DESCRIPTION

```diff
@@ -20,5 +20,7 @@ BugReports: https://github.com/as4378/opart/issues
 Suggests: 
     knitr,
     rmarkdown,
-    testthat
+    testthat,
+    dplyr,
```

Comment:
what do you use dplyr, knit, markdown for? if you don't use them please remove

Reply:
Hi @tdhock , knitr and markdown are required for Rmd file in vignette. I used dplyr when I was reading data from csv. I can remove this as now I am reading data from neuroblastoma package directly.